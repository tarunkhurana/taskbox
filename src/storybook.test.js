// storybook.test.js

// eslint-disable-next-line import/no-extraneous-dependencies
import initStoryshots from '@storybook/addon-storyshots';

initStoryshots();
