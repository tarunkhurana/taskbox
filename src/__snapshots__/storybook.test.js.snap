// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Example/Task Archived 1`] = `
<div
  className="list-item TASK_ARCHIVED"
>
  <label
    className="checkbox"
  >
    <input
      defaultChecked={true}
      disabled={true}
      name="checked"
      type="checkbox"
    />
    <span
      aria-hidden="true"
      className="checkbox-custom"
      onClick={[Function]}
    />
  </label>
  <div
    className="title"
  >
    <input
      placeholder="Input title"
      readOnly={true}
      type="text"
      value="Test Task"
    />
  </div>
  <div
    aria-hidden="true"
    className="actions"
    onClick={[Function]}
  />
</div>
`;

exports[`Storyshots Example/Task Default 1`] = `
<div
  className="list-item TASK_INBOX"
>
  <label
    className="checkbox"
  >
    <input
      defaultChecked={false}
      disabled={true}
      name="checked"
      type="checkbox"
    />
    <span
      aria-hidden="true"
      className="checkbox-custom"
      onClick={[Function]}
    />
  </label>
  <div
    className="title"
  >
    <input
      placeholder="Input title"
      readOnly={true}
      type="text"
      value="Test Task"
    />
  </div>
  <div
    aria-hidden="true"
    className="actions"
    onClick={[Function]}
  >
    <a
      aria-hidden="true"
      onClick={[Function]}
    >
      <span
        className="icon-star"
      />
    </a>
  </div>
</div>
`;

exports[`Storyshots Example/Task Pinned 1`] = `
<div
  className="list-item TASK_PINNED"
>
  <label
    className="checkbox"
  >
    <input
      defaultChecked={false}
      disabled={true}
      name="checked"
      type="checkbox"
    />
    <span
      aria-hidden="true"
      className="checkbox-custom"
      onClick={[Function]}
    />
  </label>
  <div
    className="title"
  >
    <input
      placeholder="Input title"
      readOnly={true}
      type="text"
      value="Test Task"
    />
  </div>
  <div
    aria-hidden="true"
    className="actions"
    onClick={[Function]}
  >
    <a
      aria-hidden="true"
      onClick={[Function]}
    >
      <span
        className="icon-star"
      />
    </a>
  </div>
</div>
`;

exports[`Storyshots Example/TaskList Default 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div
    className="list-items"
  >
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 1"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 2"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 3"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 4"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 5"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 6"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Example/TaskList Empty 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div
    className="list-items"
  >
    <div
      className="wrapper-message"
    >
      <span
        className="icon-check"
      />
      <div
        className="title-message"
      >
        You have no tasks
      </div>
      <div
        className="subtitle-message"
      >
        Sit back and relax
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Example/TaskList Loading 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div
    className="list-items"
  >
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
    <div
      className="loading-item"
    >
      <span
        className="glow-checkbox"
      />
      <span
        className="glow-text"
      >
        <span>
          Loading
        </span>
         
        <span>
          cool
        </span>
         
        <span>
          state
        </span>
      </span>
    </div>
  </div>
</div>
`;

exports[`Storyshots Example/TaskList With Pinned Tasks 1`] = `
<div
  style={
    Object {
      "padding": "3rem",
    }
  }
>
  <div
    className="list-items"
  >
    <div
      className="list-item TASK_PINNED"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 6(Pinned)"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 1"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 2"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 3"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 4"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
    <div
      className="list-item TASK_INBOX"
    >
      <label
        className="checkbox"
      >
        <input
          defaultChecked={false}
          disabled={true}
          name="checked"
          type="checkbox"
        />
        <span
          aria-hidden="true"
          className="checkbox-custom"
          onClick={[Function]}
        />
      </label>
      <div
        className="title"
      >
        <input
          placeholder="Input title"
          readOnly={true}
          type="text"
          value="Task 5"
        />
      </div>
      <div
        aria-hidden="true"
        className="actions"
        onClick={[Function]}
      >
        <a
          aria-hidden="true"
          onClick={[Function]}
        >
          <span
            className="icon-star"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;
